index.js

```diff
@@ -31,6 +32,22 @@ const argv = require('yargs')
         desc: 'Print an HTML file or URL to PDF',
         builder: {
             ...commonOptions,
+            'emulatemedia': {
```

Comment:
rename to `emulate-media` to be consistent with other multi-term options (will materialize as `argv.emulateMedia` then)

Reply:
done